docs/help/Coding-Style-Guideline.md

```diff
@@ -160,47 +157,20 @@ Use either // or /* */ style comments. However, // style is much prefered.
 
 # Coding Style Guideline for Javascript
 
-This coding standard is based on [google javascript coding standard](https://google.github.io/styleguide/javascriptguide.xml)
+The Javascript code follows [AirBnB guideline](https://github.com/airbnb/javascript)
```

Comment:
@kolipka It looks URL(https://github.com/airbnb/javascript) which you link has rules for ES6.